### ISSUE TITLE
Hidden config file on Windows breaks the application.

### ISSUE THREAD

**Issue by seep** [sqlectron/sqlectron-gui#265]
This is an odd one. Marking the `.sqlectron.json` file as hidden in Windows prevents the GUI from reading/parsing/displaying it's saved connections, and from creating new connections. I'm not sure where the error is happening, and nothing is logged to the console in the dev tools. I also tried poking around with Node a little to see if the `fs.stat` signature changes or returns an error for hidden files but I didn't notice anything. I'm using Sqlectron 1.16.0 on Windows 10.

**Comment by maxcnunes** [comment]
Yeah that is weird. I will try to check it out once I have some free time. Thanks for reporting the issue.

**Comment by maxcnunes** [comment]
I thought I would have time to investigate it. But I don't :( If someone else has interest in doing that. Please do.
